robot/markers.py

```diff
@@ -1,30 +1,15 @@
 import math
-
+from collections import namedtuple
 from robot.game_specific import TOKEN, WALL
+from typing import NewType, Tuple, List
 
+TMetres = NewType('TMetres', float)
+TDegrees = NewType('TDegrees', float)
+TRadians = NewType('TRadians', float)
+TPixel = NewType('TPixel', float)
+TPixelCoordinates = NewType('TPixelCoordinates', Tuple[TPixel, TPixel])
 
-class CartCoord:
-    """Represents a cartesian co-ordinate point."""
-
-    def __init__(self, x, y, z):
-        self._x = x
-        self._y = y
-        self._z = z
-
-    @property
-    def x(self):
-        """X co-ordinate of the cartesian position."""
-        return self._x
-
-    @property
-    def y(self):
-        """Y co-ordinate of the cartesian position."""
-        return self._y
-
-    @property
-    def z(self):
-        """Z co-ordinate of the cartesian position."""
-        return self._z
+CartCoord = namedtuple("CardCoord", ["x", "y", "z"])
```

Comment:
While the syntax (at least before Python 3.6) is ugly, I think it would be worth this being a `typing.NamedTuple`:

``` python
CartCoord = NamedTuple('CartCoord', [
    ('x', int),
    ('y', int),
    ('z', int),
])
```

Reply:
Resolved in #59 by 1d04e79c2a52d814bdef88c0469a38931a58b079.